flake.nix

```diff
@@ -56,6 +56,7 @@
                       vars = with hostConfig.config.lollypops; {
                         REMOTE_USER = deployment.user;
                         REMOTE_HOST = deployment.host;
+                        REBUILD_ACTION = deployment.rebuild-action;
```

Comment:
Having `rebuild-action` in configuration feels weird tbh.
I think it's enough to do something like this: `REBUILD_ACTION = "{{default \"switch\" .REBUILD_ACTION}}";`
This should allow running `REBUILD_ACTION=dry-activate nix run . -- <hostname>` if needed.

Reply:
Agreed, this is a good way to do it.